src/style.css

```diff
@@ -0,0 +1,85 @@
+:root {
+  --border-color: #e1e7ed;
+  --bg-color: #fff;
+  --text-color: #3d4e61;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+.container {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  margin: 0;
+}
+
+.search-bar__input {
+  font-family: "Avenir", sans-serif;
+  font-weight: 400;
+  box-sizing: border-box;
```

Comment:
set box-sizing for all elements or remove it

Reply:
I deleted it